coding/zip_creator.cpp

```diff
@@ -129,13 +74,25 @@ bool CreateZipFromFiles(std::vector<std::string> const & files, std::string cons
     return false;
 
   auto const compressionLevel = GetCompressionLevel(compression);
-  zip::FileInfo const fileInfo = {};
+  zip::FileInfo fileInfo = {};
+  FillZipLocalDateTime(fileInfo.tmz_date);
 
   try
   {
-    for (auto const & filePath : files)
+    int suffix = 0;
+    for (size_t i = 0; i < files.size(); i++)
     {
-      if (zip::Code::Ok != zip::OpenNewFileInZip(zip.Handle(), filePath, fileInfo, "",
+      auto const & filePath = files.at(i);
+      std::string fileInArchive = filesInArchive.at(i);
+      if (!strings::IsASCIIString(fileInArchive))
+      {
+        if (suffix == 0)
+          fileInArchive = "OrganicMaps.kml";
+        else
+          fileInArchive = "OrganicMaps_" + std::to_string(suffix) + ".kml";
+        suffix++;
```

Comment:
```suggestion
          fileInArchive = "OrganicMaps_" + std::to_string(suffix++) + ".kml";
```

Reply:
It will not increment for suffix == 0 so it will stay on 0?

Reply:
You're right!